.github/CODEOWNERS

```diff
@@ -0,0 +1,33 @@
+# Automatically request reviews from maintainers
+
+# Package
+/pybamm/discretisations/ @martinjrobins @rtimms @tinosulzer
+/pybamm/experiment/ @brosaplanella @martinjrobins @rtimms @tinosulzer @TomTranter
+/pybamm/expression_tree/ @martinjrobins @rtimms @tinosulzer
+/pybamm/geometry/ @martinjrobins @rtimms @tinosulzer
+/pybamm/input/ @brosaplanella @DrSOKane @rtimms @tinosulzer @TomTranter @kratman
+/pybamm/meshes/ @martinjrobins @rtimms @tinosulzer @rtimms
+/pybamm/models/ @brosaplanella @DrSOKane @rtimms @tinosulzer @TomTranter @rtimms
+/pybamm/parameters/ @brosaplanella @DrSOKane @rtimms @tinosulzer @TomTranter @rtimms  @kratman
+/pybamm/plotting/ @martinjrobins @rtimms @Saransh-cpp @tinosulzer @rtimms  @kratman
+/pybamm/solvers/ @martinjrobins @rtimms @tinosulzer @TomTranter @rtimms
+/pybamm/spatial_methods/ @martinjrobins @rtimms @tinosulzer @rtimms
+/pybamm/* @pybamm-team/maintainers # the files directly under /pybamm/, will not recurse
+
+# CI/CD workflows
+/.github/ @martinjrobins @Saransh-cpp @agriyakhetarpal @kratman @arjxn-py
+
+# Benchmarks
+/benchmarks/ @brosaplanella @Saransh-cpp @agriyakhetarpal @arjxn-py
+
+# Documentation
+/docs/ @pybamm-team/maintainers
+
+# Example scripts
+/examples/ @pybamm-team/maintainers
```

Comment:
Ah, I did this because I could not add @agriyakhetarpal to the file back then. I think the best person to maintain or own these folders would be @agriyakhetarpal, if he is comfortable doing so.

Reply:
You can add me. I am not bothered by notifications

Reply:
You can also consider me adding for `/docs/` atleast. Nor am I bothered by notifications 🙂

Reply:
Me three. I am not bothered by notifications either :) I would be comfortable maintaining both `docs/` and `examples/`.

Reply:
```suggestion
/docs/ @kratman @arjxn-py @agriyakhetarpal 

# Example scripts
/examples/ @kratman @agriyakhetarpal 
```